Volume/Views/BookmarksList.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  BookmarksList.swift
+//  Volume
+//
+//  Created by Cameron Russell on 11/15/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct BookmarksList: View {
+    var body: some View {
+        GeometryReader { geometry in
+            NavigationView {
+                ScrollView(showsIndicators: false) {
+                    Header(text: "Saved Articles")
+                    if let savedArticles = articleData.filter{ $0.saved }, savedArticles.count > 0 {
+                        ForEach (savedArticles) { article in
+                            ArticleRow(article: article)
+                                .padding([.bottom, .leading, .trailing])
+                        }
+                    } else {
+                        VStack {
+                            Spacer(minLength: geometry.size.height / 5)
+                            VolumeMessage(message: .noBookmarks)
+                        }
```

Comment:
We should try to avoid `GeometryReader` where possible. Instead of using that here, why not put a spacer above and a below `VolumeMessage` to get it in the middle?

Reply:
While I agree `GeometryReader` should be avoid, as it's been somewhat buggy/yielding unexpected results when I've played around with it, SwiftUI does not make the desired behavior (simply pinning the view to the center of the screen) very easy. `GeometryReader` also responds to orientation changes if that's something we need later down the line, who knows